pages/docs/index.js

```diff
@@ -251,17 +251,15 @@ export default function DocsIndex() {
       <Page>
         <h1 className="text-4xl mb-4 font-bold">Documentation</h1>
         <p>
-          Here you should find everything you need from getting started with
+          Here you should find everything you need, from getting started with
           creating your Profile to more advanced topics. You can contribute to
-          our documentation and find the files here{" "}
+          our documentation and find the{" "}
           <a
             href="https://github.com/EddieHubCommunity/LinkFree/tree/main/pages/docs"
             target="_blank"
             rel="noreferrer"
             className="text-sky-700 break-words"
-          >
-            https://github.com/EddieHubCommunity/LinkFree/tree/main/pages/docs
-          </a>
+          >markdown files</a>
```

Comment:
I think Markdown should be capitalized, IMO.
```suggestion
          >Markdown files</a>
```